addons/web/static/tests/core/hotkeys/hotkey_service_tests.js

```diff
@@ -527,3 +487,35 @@ QUnit.test("registrations and elements belong to the correct UI owner", async (a
     comp1.destroy();
     comp2.destroy();
 });
+
+QUnit.test("replace the overlayModifier", async (assert) => {
+    assert.expect(3);
+
+    const hotkeyService = serviceRegistry.get("hotkey");
+    patchWithCleanup(hotkeyService, {
+        overlayModifier: "alt+shift",
+    });
```

Comment:
I am 100% ok with patching here, but I am asking myself if it wouldn´t be "safer" (in regards of people that oftentimes look at tests as a documentation point) to do a proper service extension ?

```suggestion
// import hotkeyService descriptor from hotkey_service.js
serviceRegistry.add("hotkey", { ...hotkeyService, overlayModifier: "alt+shift" }, { force:true });
```